README.md

```diff
@@ -308,4 +308,44 @@ There may only be up to 1 dead pile, marked by the Death Token,
 which must be placed by the player responsible for the pile's death.
 When a pile dies, the token moves to the newly dead pile, and any previously dead pile is revived.
 
+## Veto
+_Added by Jonathan, 8 Nov 2023_
 
+Each player has one veto.
+When the Active Player plays on the pile, any player may veto.
+The Active Player must then retrieve the cards they played, and make a different move.
+The vetoing player's veto is expended.
+
+## Starting Salad
+_Added by Matt, 8 Nov 2023_
+
+After dealing, but before the round begins,
+the Dealer must distributed one card, face up, to each player,
+starting from the Dealer's left.
+This is the player's Salad.
+The player with the lowest (Aces low) Salad plays first.
+
+Play continues clockwise.
+
+As long as there is no player with the lowest top card in their Salad,
+the player with the top Salad card receives an additional card on their Salad.
```

Comment:
you mean "with the lowest salad"?

Reply:
It was supposed to be "oldest top salad card"